client/js/lounge.js

```diff
@@ -89,6 +89,13 @@ $(function() {
 				text: target.data("title"),
 				data: target.data("target"),
 			});
+			if (target.hasClass("lobby")) {
+				output += templates.contextmenu_item({
+					class: "list",
+					text: "Channel list",
```

Comment:
I'm debating if it would not be more user-friendly to have something like "List channels on this server" or something.

Reply:
I think that’s too long... maybe “List existing channels”?

Reply:
I agree, it's too long.

Maybe "List Channels" or "List all channels" or "List of Channels" or something.

Reply:
I don't really see whats wrong with Channel List, its pretty straight forward. List Channels would fit better to disconnect tho since both are an action.

Reply:
I guess the issue with "Channel List" or "List Channels" is ... what channels? But none of our solutions fix that problem, really. I'd be fine with any of these other than "List channels on this server"

Reply:
<img width="250" alt="screen shot 2017-12-06 at 18 36 45" src="https://user-images.githubusercontent.com/113730/33691145-ca42aa52-dab4-11e7-8497-4a0fc0a0f71f.png">

:man_shrugging: 

Let's go with "List all channels" then.